packages/services/src/services/Translation/__tests__/Translation.test.js

```diff
@@ -76,7 +98,69 @@ describe('TranslationAPI', () => {
         origin: 'https://ibm.com',
       },
     });
-
+    delete response['id'];
     expect(response).toEqual(responseSuccess);
   });
+
+  it('should return a json with a recent timestamp', async () => {
+    const mockDate = 1546300800000; // Epoch time of January 1, 2019 midnight UTC
+    global.Date.now = jest.fn(() => mockDate);
+
+    // using very old cached session
+    sessionStorageMock.setItem(
+      'dds-translation-us-en',
+      JSON.stringify(oldSession)
+    );
+
+    const previousSession = JSON.parse(
+      sessionStorageMock.getItem('dds-translation-us-en')
+    );
+
+    expect(previousSession.id).toEqual('TRANSLATION_FRESH');
+
+    const response = await TranslationAPI.getTranslation({
+      lc: 'en',
+      cc: 'us',
+    });
+
+    const newSession = JSON.parse(
+      sessionStorageMock.getItem('dds-translation-us-en')
+    );
+
+    response['id'] = newSession.id;
+
+    // newest response and storage data should match
+    expect(response).toEqual(newSession);
+
+    // should contain timestamp
+    expect(response).toHaveProperty('timestamp');
+
+    // should equal mock timestamp
+    expect(response.timestamp).toEqual(mockDate);
+    expect(response.id).toEqual('TRANSLATION_FRESH');
+  });
+
+  it('timestamp should not change if within two hours', async () => {
+    // using recent cached session
+    sessionStorageMock.setItem(
+      'dds-translation-us-en',
+      JSON.stringify(responseSuccess)
+    );
+
+    const previousSession = JSON.parse(
+      sessionStorageMock.getItem('dds-translation-us-en')
+    );
+
+    const response = await TranslationAPI.getTranslation({
+      lc: 'en',
+      cc: 'us',
+    });
+
+    // body and timestamp should remain unchanged
+    expect(response).toEqual(previousSession);
+  });
```

Comment:
Given we don't have any code to change the timestamp of cache, this test case seems not necessary:

```suggestion
```